src/request/remove-key/RemoveKey.js

```diff
@@ -0,0 +1,150 @@
+/* global ExportWords */
+/* global ExportFile */
+/* global PassphraseBox */
+/* global Nimiq */
+/* global KeyStore */
+class RemoveKey {
+    /**
+     * if a complete page is missing it will be created.
+     * However these pages wil be the default pages which usually don't match the applications requirements.
+     * Refer to the corresponsing _buildRemoveKey as well as
+     * the Build functions of ExportWords and ExportFile to see the general Structure.
+     * @param {KeyguardRequest.ParsedSimpleRequest} request
+     * @param {Function} resolve
+     * @param {Function} reject - 'keyId not found','Unknown mnemonic type','Unsupported type','Rounds out-of-bounds'
+     */
+    constructor(request, resolve, reject) {
+        this._resolve = resolve;
+        this._request = request;
+        this._reject = reject;
+
+        // reject cases are actual errors so will be reject cases for the entire request.
+        this._exportWordsHandler = new ExportWords(request, this.run.bind(this), this._reject.bind(this));
+        this._exportFileHandler = new ExportFile(request, this.run.bind(this), this._reject.bind(this));
+
+        /** @type {HTMLElement} */
+        const $removeKey = (
+            document.getElementById(RemoveKey.Pages.REMOVE_KEY)
+                ? document.getElementById(RemoveKey.Pages.REMOVE_KEY)
+                : this._buildRemoveKey()
+        );
```

Comment:
I feel this can be simplified to
```javascript
const $removeKey = document.getElementById(RemoveKey.Pages.REMOVE_KEY) || this._buildRemoveKey();
```